packages/angular/cli/lib/config/schema.json

```
@@ -67,7 +67,7 @@
           }
         },
         "analytics": {
-          "type": "boolean",
+          "type": "string",
```

```suggestion
          "type": ["boolean", "string"],
```
It can actually be either type.